### ISSUE TITLE
how to run swagger in windows : getting err:-  bash: swagger: command not found

### ISSUE THREAD

**Issue by kundhu2367** [go-swagger/go-swagger#1973]
## Problem statement

while running swagger I get the below error
```swagger generate spec -o ./swagger.json --scan-models
bash: swagger: command not found```

I Installed the go-swagger package by `go get -u github.com/go-swagger/go-swagger/cmd/swagger`


## Swagger specification

## Steps to reproduce

## Environment
swagger version: x.x.x  
go version: 1.11
OS:   windows

**Comment by fredbi** [comment]
Maybe `go install -a ...`?

**Comment by casualjim** [comment]
Are you able to run other binaries after installing them with `go get`
You may need to modify your `PATH` variable with `%GOPATH%\bin`

**Comment by kundhu2367** [comment]
No, this is the first binary I'm trying to run. I did add GOPATH\bin to my path.

**Comment by dre2004** [comment]
Do you have multiple directories in GOPATH? The default location is %USERPROFILE%\go but if you've got multiple directories in GOPATH (I have %USERPROFILE%\go & C:\code\go) you may have to explicitly add both directories to path (eg for me its %USERPROFILE%\go & C:\code\go\bin).

Also depending on which path you have set (system wide or user specific) you may need to restart (system wide needs a restart). It's probably easier to update just the user path.

**Comment by amitNinjax** [comment]
![swagger](https://user-images.githubusercontent.com/63649130/97256105-3288e680-1838-11eb-9b09-c2883733601d.png)
Facing the same problem.
Can anyone help me quickly

**Comment by xclidongbo** [comment]
mac  the same error
